### ISSUE TITLE
Request: Admins can reset other peoples passwords.

### ISSUE THREAD

**Issue by daanbreur** [Wingysam/Christmas-Community#129]
This year I had the problem that some people have forgotten the password they set last year. So sadly I had to instruct them to make a new account and help them manually move over every single item on their list including metadata. 
It would be nice if admin accounts had a button to either:
- generate a password reset link
- set a new (temporary) password

**Comment by Wingysam** [comment]
They do
![image](https://github.com/Wingysam/Christmas-Community/assets/18403742/d14c29f0-e57e-431c-ba2a-b67d3ffd446e)

**Comment by Wingysam** [comment]
Maybe there's a UI discoverability problem here? What can be changed to help admins find the button?

**Comment by daanbreur** [comment]
Wait... WHAT? You don't know how long I searched for it.

I think the admin GUI does need a bit of a UI overhaul. Maybe make a table with all users instead of seperate pages. I can make a small mockup of what I mean later.

**Comment by Wingysam** [comment]
Yeah it's in the tiny blue pencil on a notepad icon next to each user. I can see how admins wouldn't find it. Please do send a mockup of what a better UI would look like!

**Comment by cj13579** [comment]
This is pretty simple, but might achieve what's being sought after here?

![image](https://github.com/user-attachments/assets/68daa6b5-4c25-4c60-9696-4a277bae06c3)

Let me know if you'd like me to make a PR with this change?
